libraries/classes/DatabaseInterface.php

```diff
@@ -2282,6 +2273,29 @@ public function isPercona(): bool
         return $this->isPercona;
     }
 
+    /**
+     * Set version
+     *
+     * @param array $version Database version information
+     * @phpstan-param array<array-key, mixed> $version
+     */
+    public function setVersion(array $version): void
+    {
+        $this->versionString = $version['@@version'] ?? '';
+        $this->versionInt = Utilities::versionToInt($this->versionString);
+        $this->versionComment = $version['@@version_comment'] ?? '';
+
+        if (stripos($this->versionString, 'mariadb') !== false) {
+            $this->isMariaDb = true;
```

Comment:
```suggestion
            $this->isMariaDb = true;
            return;
```
If isPercona is set to false by default we can quick exit here

Reply:
Thank you to comment on this code, I think should it refresh value both `isMariaDb` and `isPercona` .. so remove condition check is better.